pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1005,7 +1007,7 @@ func (a *ApplicationBackupController) uploadCRDResources(backup *stork_api.Appli
 				continue
 			}
 			crdName := ruleset.Pluralize(strings.ToLower(v.Kind)) + "." + v.Group
-			res, err := apiextensions.Instance().GetCRD(crdName, metav1.GetOptions{})
+			res, err := apiextensions.Instance().GetCRDV1beta1(crdName, metav1.GetOptions{})
```

Comment:
Shouldn't we get v1 CRD as well if v1beta1 is not found ?

Reply:
yes, thanks for catch